packages/round-manager/src/features/common/FormDropDown.tsx

```diff
@@ -0,0 +1,129 @@
+import { FC } from "react";
+import {
+  Control,
+  useController,
+  UseFormRegisterReturn,
+  FieldErrors,
+  FieldPath,
+} from "react-hook-form";
+import { Listbox, Transition } from "@headlessui/react";
+import { CheckIcon, SelectorIcon } from "@heroicons/react/solid";
+import { Round } from "../api/types";
+import _ from "lodash";
+import { classNames } from "common";
+import { Fragment } from "react";
+
+export const FormDropDown: FC<{
+  register: UseFormRegisterReturn<string>;
+  errors: FieldErrors<Round>;
+  control: Control<Round>;
+  label: string;
+  id: FieldPath<Round>;
+  options: string[];
+  defaultValue: string;
+}> = ({ errors, control, label, id, options, defaultValue }) => {
+  // @ts-expect-error "required" appears on the error type, not sure where from
+  const errorMessage = _.get(errors, id)?.message;
```

Comment:
qq: what are the benefits of using lodash instead of directly doing `const errorMessage = errors[id]?.message;`? I don't get the "required" error either if I try that locally, so we might be able to remove the `@ts-expect-error` comment.

Reply:
Alternatively, can we `import { get } from "lodash";` like in `FormInputField` instead of importing everything? (I still prefer the solution above if possible though)